modules/custom/openy_group_schedules/templates/groupex-individual-location-links.html.twig

```diff
@@ -0,0 +1,31 @@
+{#
+/**
+ * @file
+ * Template file for individual location schedule page block.
+ *
+ * Available variables:
+ * - branch_id: GroupEx branch id.
+ */
+#}
+
+<h2>Register for Classes and Programs</h2>
```

Comment:
I'm wondering if we should have this template here, because it's not related to Personify.